forms/ui/moc_DrawArea.cpp

```diff
@@ -1,90 +0,0 @@
-/****************************************************************************
```

Comment:
This and the files above were deleted as now we use the moc compiler, is that correct?

Reply:
Exactly.